### PR TITLE
Access to ecosystem from go code

### PR DESCRIPTION
**Pull request by skillful-alex**
Now, to access the ecosystem, you need to [include a list of servers in the code.](https://github.com/prysmaticlabs/prysm/blob/master/shared/roughtime/roughtime.go#L30) or distribute the json file next to the binary. **This is a way to obsolete the list of servers.**

The PR provides an way to update the list of ecosystem servers from the https://github.com/cloudflare/roughtime/ repository, and easy use it: `Do (roughtime.Ecosystem, *dialAttempts, *dialTimeout, nil)
`

### CODE REVIEW

**Comment by kivutar** [comment]
You could add a `go generate` instruction somewhere in the main library, so that users just have to run `go generate` instead of `go run ./ecosystem_json_go_builder`.

**Comment by skillful-alex** [comment]
> You could add a `go generate` instruction somewhere in the main library, so that users just have to run `go generate` instead of `go run ./ecosystem_json_go_builder`.

cool! done

File: README.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ should do the following:
      3. where in the world your server is located; and
      4. whether you will guarantee up time, and if so, how you will do so.
 
+  * Add your server's configuration to `ecosystem.json.go`. For this, it is
+    preferable to use the "go generate" command.
+
```
Comment (wbl) [inline README.md]: You might want to mention that this is generated from ecosystem.json so the edit should be made there.

**Comment by skillful-alex** [comment]
Changed readme. If my English is bad please correct for the correct text.

**Comment by wbl** [comment]
Thanks for the contribution!

**Pull request merged**